src/extension.ts

```diff
@@ -433,7 +433,7 @@ class ConventionalCommitMessage {
   private static allowBreakingChanges(
     czConfig: CzConfig | undefined,
     selectedType: string | undefined
-  ): boolean {
+  ): czConfig is CzConfig {
```

Comment:
Oh, I think you reverted in the wrong line, right?

Reply:
how clumsy of me! OMG

Reply:
I have reverted my change in there